packages/datadog-plugin-cookie-parser/test/integration-test/client.spec.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+const {
+  createSandbox, varySandbox, curl,
+  FakeAgent, spawnPluginIntegrationTestProc
+} = require('../../../../integration-tests/helpers')
+const { assert } = require('chai')
+const { withVersions } = require('../../../dd-trace/test/setup/mocha')
+
+withVersions('cookie-parser', 'cookie-parser', version => {
+  describe('ESM', () => {
+    let sandbox, variants, proc, agent
+
+    before(async function () {
+      this.timeout(50000)
+      sandbox = await createSandbox([`'cookie-parser@${version}'`, 'express'], false,
+        ['./packages/datadog-plugin-cookie-parser/test/integration-test/*'])
+      variants = varySandbox(sandbox, 'server.mjs', 'cookie-parser', undefined, 'cookieParser')
+    })
+
+    after(async function () {
+      this.timeout(50000)
+      await sandbox.remove()
+    })
+
+    beforeEach(async () => {
+      agent = await new FakeAgent().start()
+    })
+
+    afterEach(async () => {
+      proc && proc.kill()
+      await agent.stop()
+    })
+
+    for (const variant of varySandbox.VARIANTS) {
+      it(`is instrumented loaded with ${variant}`, async () => {
```

Comment:
you're not really testing that it's instrumented right ?
If the instrumentation was completely disabled this test would still pass no ?